src/day-state-manager.ts

```diff
@@ -3,10 +3,10 @@ const {toMarkingFormat} = require('./interface');
 
 
 export function getState(day: XDate, current: XDate, props: any) {
-  const {minDate, maxDate, disabledByDefault, context} = props;
+  const {minDate, maxDate, disabledByDefault, context, disableDaySelection} = props;
```

Comment:
Why is `disableDaySelection` part of the 'props' if it's not a prop? It's confusing, why not add an option object or another pharam?

Reply:
i agree, moved to be a different parameter